versions.properties

```diff
@@ -119,3 +119,5 @@ version.org.spekframework.spek2..spek-runtime-jvm=2.0.13
 version.retrofit2=2.9.0
 
 version.sqldelight=1.4.4
+
```

Comment:
`./gradlew refreshVersions` sorted it like that. 🤔

Reply:
It's a known issue, will be fixed in the next version :)